cognite/client/data_classes/transformations/common.py

```diff
@@ -306,9 +355,10 @@ def _load_destination_dct(
     if destination_type in simple:
         return simple[destination_type](**snake_dict)
 
-    nested: Dict[str, Union[type[Nodes], type[Edges]]] = {
+    nested: Dict[str, Union[type[Nodes], type[Edges], type[Instances]]] = {
```

Comment:
You may consider using the pipe operator for unions:
```suggestion
    nested: Dict[str, type[Nodes] | type[Edges] | type[Instances]] = {
```